control/config.go

```diff
@@ -27,12 +27,34 @@ import (
 	"strconv"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
 
 	"github.com/intelsdi-x/snap/core"
 	"github.com/intelsdi-x/snap/core/cdata"
 	"github.com/intelsdi-x/snap/core/ctypes"
 )
 
+type flagConfig struct {
```

Comment:
Looks good.  If you move this struct and its three methods to to pkg (e.g. pkg/cli/flags.go) I think we can get it merged.